test/acceptance/features/steps/kubernetesapi.py

```diff
@@ -114,6 +114,13 @@ def registered_service_in_catalog(rs_name, catalog):
     return False
 
 
+def catalog_is_empty(catalog):
+    if "spec" in catalog:
+        if catalog["spec"] == {}:
+            return True
+    return False
```

Comment:
this may be simplified

```suggestion
    return "spec" in catalog and catalog["spec"] == {}
```